packages/toolpad-app/src/toolpad/AppEditor/AppEditorShell.tsx

```diff
@@ -138,13 +142,25 @@ export interface ToolpadShellProps {
 
 export default function AppEditorShell({ appId, ...props }: ToolpadShellProps) {
   const domLoader = useDomLoader();
+  const releases = client.useQuery('findLastRelease', [appId]);
```

Comment:
Should we display the error state?

Reply:
I'm thinking that maybe not? This is not a critical feature and if there is an error IMO it could be handled gracefully without even letting user know about that 🤔

Reply:
Let's show a message in the dropdown when `releases.error`.
I'd also rename "releases" to something else, it doesn't really contain multiple releases.